gravityflow/gravityflow
# Fix long name steps to display correctly on workflow details page

## Description
This PR fixes an issue with Workflow Status Box on the Workflow details page, where a long name Workflow Step would not display correctly.

This resolves [gravityflow/backlog#164](https://github.com/gravityflow/backlog/issues/164).

## Testing instructions
1. Create a workflow step for a form, such that the workflow step has a long name (e.g. Approval for Project Manager).
2. Process an entry for this workflow, note how the workflow step status box appears.
3. Switch to this branch, and repeat step 2. Confirm the appearance is now fine.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
